test/everypolitician_test.rb

```diff
@@ -195,4 +195,13 @@ def test_expose_the_statement_count_of_a_legislature
     end
   end
 
+  def test_lastmod_is_a_time
+    VCR.use_cassette('countries_json') do
+      australian_senate = Everypolitician.country(slug: 'Australia').legislature(slug: 'Senate')
+      assert_equal 2016, australian_senate.lastmod.year
+      assert_equal 7, australian_senate.lastmod.month
+      assert_equal 24, australian_senate.lastmod.day
+    end
+  end
+
```

Comment:
Almost there, but although this is claiming to test that it's _time_, the tests are actually only testing that it's a _date_. Adding a test for `lastmod.hour` would make sure it's actually a date, and should also ensure we don't ever introduce a regression on the gmtime issue you uncovered earlier, which these tests currently wouldn't.